pkg/proxy/metrics/metrics.go

```diff
@@ -264,3 +275,40 @@ func RegisterMetrics() {
 func SinceInSeconds(start time.Time) float64 {
 	return time.Since(start).Seconds()
 }
+
+var _ metrics.StableCollector = &ctStateInvalidPacketsCollector{}
```

Comment:
I´m not familiar with this stuff, @dgrisonnet do you mind to take a look to the metrics part?

Reply:
oh it seems was already reviewed by instrumentation https://github.com/kubernetes/kubernetes/pull/122812#discussion_r1499624247

Reply:
yup.

Reply:
This also removed the metric sync dependency on proxier sync loop 😃